packages/tools/devtools/devtools-core/src/DevtoolsLogger.ts

```diff
@@ -70,6 +71,11 @@ export class DevtoolsLogger extends TelemetryLogger {
 			this.postLogHistory();
 			return true;
 		},
+		[CloseDevtools.MessageType]: (untypedMessage) => {
```

Comment:
This shouldn't be necessary. We just want to make sure the view purges any stale state when the devtools are disposed. This lifetime of this object's logs can match the lifetime of the object itself - that shouldn't be a problem.

Reply:
If two `FluidDevtools` instances happen to share a `DevtoolsLogger` instance, that should be okay.

Reply:
I tested out on not reset the _telemetryLog list. Once the reload finished, telemetry log page will receive all logs in `TELEMETRY_HISTORY` message. It will behave as the same as previously that  we simply add the new events from the subsequent session to the view. Shouldn't the telemetry view refresh with page?

Reply:
Are you testing this be refreshing the webpage? If we refresh the webpage, we will be replacing this object altogether. So long as we clear out the list of events being stored by React's local state on the view side, I think we should be okay.

Reply:
I see! Then remove the reset cache in `DevtoolsLogger`